tests/FSharp.Data.Tests/Http.fs

```diff
@@ -129,23 +192,31 @@ let ``Cookies is not added in cookieContainer but is still returned when addCook
 
 [<Test>]
 let ``Web request's timeout is used`` () =
-    let exc = Assert.Throws<WebException> (fun () ->
-        Http.Request("http://httpstat.us/200?sleep=1000", customizeHttpRequest = (fun req -> req.Timeout <- 1; req)) |> ignore)
-    Assert.AreEqual(typeof<TimeoutException>, exc.InnerException.GetType())
+    let localServer = startHttpLocalServer()
+    using localServer <| fun _ ->
+        let exc = Assert.Throws<WebException> (fun () ->
+            Http.Request(localServer.BaseAddress + "/200?sleep=1000", customizeHttpRequest = (fun req -> req.Timeout <- 1; req)) |> ignore)
+
+        Assert.That(exc.Status, Is.EqualTo(WebExceptionStatus.Timeout))
```

Comment:
Any reason why `Assert.AreEqual` can't still be used?

Reply:
Not really. I find this syntax more easy to read, that's all (There is less ambiguity which value is the expected value and which one is the actual value).  

I can change it to `Assert.AreEqual` if you feel like consistency with the rest of the tests is more important.

Reply:
Yes, please change it so that it's consistent with the rest.

Reply:
Ok. (actually, the rest of the file uses `... |> should equal ...` so I'm going to do the same)